### DISCUSSION TITLE
Cannot display the top page properly.

### DISCUSSION THREAD

**Discussion by TaisukeMorimoto55** [mozilla/hubs#6062] [Hubs Cloud Questions]
**Description**
I deployed hubs cloud v1.1.5, but it does not transition to the top screen properly and redirects to /admin. After being redirected to /admin, I get "Missing file admin.html. Please try again." message.

**To Reproduce**
1. Run cloud formation stack  (hubs cloud template v1.1.5)
2. Create completely cloud formation stack 
3. Access domain (which is set up on Route 53).
4. Redirect /admin
5. See error message. "Missing file admin.html. Please try again".  In develop tool console, "Failed to load resource: the server responded with a status of 503 ()"

**Expected behavior**
1. Top screen is displayed correctly.

**Screenshots**
- https://{domain}/admin access errror message
![image](https://user-images.githubusercontent.com/105335052/235042400-b6339e00-58e1-4211-a7f1-a0c4bd156d0b.png)

- Checking target groups, the app instance is unhealthy.
![image](https://user-images.githubusercontent.com/105335052/235042140-106d5f34-6e23-4aba-99cc-53a0a4c99b9c.png)

- I connect to the app instance via ssh and run "journalctl -f" and get the following error log
1. Log of access to domain
![image](https://user-images.githubusercontent.com/105335052/235043408-3bcff59c-4904-48d7-a4e4-db189f955b25.png)
2. Log of health check
![image](https://user-images.githubusercontent.com/105335052/235043433-230d4627-a247-49f8-8b34-b1fcc1f8cb26.png)

**Hardware** 
 - Device: Desktop, phone
 - OS: Windows, mac, iOS
 - Browser: chrome, edge

**Additional context**
- The environment created in the same way in January 2023 is working fine.
- I have fixed the s3 permission error according to the work around https://github.com/mozilla/hubs/discussions/6049 . Other than that, no changes have been made to the template. No custom source code modifications or deployments.

**Comment by matthewbcool** [comment]
Thanks for providing good context @TaisukeMorimoto55.

This issue was filed on the hubs client last week, converting to hubs cloud discussion. @pattersonbl2 could you help take a look?

**Reply by TaisukeMorimoto55** [reply]
@matthewbcool  @pattersonbl2 
This error can be observed intermittently. Any solutions or tips you can think of?

**Reply by TaisukeMorimoto55** [reply]
<img width="1332" alt="image" src="https://github.com/mozilla/hubs/assets/105335052/4c8ffcc4-0afc-4be0-aa37-c82d19d02623">
It seems that some services are down.
For postgrest, I was able to confirm from the logs that login to the DB was failing.

` closing because: no such database: polycosm_production (age=0s)`
`(nodb)/(nouser)@127.0.0.1:35622 pooler error: no such database: polycosm_production`
`(nodb)/(nouser)@127.0.0.1:35622 login failed: db=polycosm_production user=postgres`
`cloud-init[1244]: psql: ERROR:  no such database: polycosm_production`

Any ideas as to why this is failing would be greatly appreciated.
